app/controllers/admin/dfc_product_imports_controller.rb

```diff
@@ -42,5 +44,24 @@ def index
       flash[:error] = e.message
       redirect_to admin_product_import_path
     end
+
+    private
+
+    def adjust_to_wholesale_price(broker, product)
+      transformation = broker.best_offer(product.semanticId)
+
+      return if transformation.factor == 1
+
+      wholesale_variant_price = transformation.offer.price
+
+      return unless wholesale_variant_price
```

Comment:
unless price is present? a number? positive? 

I guess for all intents and purposes it doesn't matter, if it's truthy, we can assume in this context that it's valid?  I'm not familiar with the price data structure so I can only assume. 

But if we're going to do a division on it later, it might we worth checking it a bit more thoroughly.

Reply:
Yes, I was aware of that assumption. Thanks for checking though. It's either a number or nil.